niworkflows/data/nipreps.json

```diff
@@ -83,11 +83,11 @@
     },
     {
       "name": "fmap",
-      "pattern": "(phasediff|magnitude[1-2]|phase[1-2]|fieldmap|epi)\\.nii"
+      "pattern": "(phasediff|magnitude[1-2]|phase[1-2]|fieldmap|epi|m0scan)\\.nii"
```

Comment:
This creates a weird pseudo-entity `"fmap": "m0scan"`, which duplicates the suffix. This quirk is unlikely to be replicated in other APIs, and I think we're better off eliminating uses of it than adding use cases to it.

Reply:
Sorry, I'm not sure I understand 100%. Are you saying that you'd prefer to recommend to dataset curators that they place reverse phase-encoded M0 scans in the `perf` folder instead of the `fmap` folder, or that I should support this on ASLPrep's side rather than in niworkflows?

Reply:
No, just that all this will do is populate an `fmap` entity in the BIDSLayout database. You already can retrieve it with `layout.get(suffix="m0scan", datatype="fmap")`.

Also, we should make it clearer that `m0scan` is a valid suffix for fieldmap files.